integration-tests/USECASES.MD

```diff
@@ -90,3 +90,8 @@ Configuration Overrides Usecases
 | --- | --- |
 | Primary Server Repick | A backup server becomes the primary server when a primary server fails|
 | HTTP Session Migration | Verify in-memory HTTP session State replication |
+
+| Server Discovery | Use Case |
+| --- | --- |
+| Discover a newly started server | stop operator and apply the modified domain.yaml with replicas count increated, restart Operator and verify that the cluster is scaled up accordingly |
+| Liveness probe | restart Operator and verify the liveness probe by killing admin server and all managed servers in the cluster | 
```

Comment:
Liveness probe is not tested in this use case. Can you correct this use case description?